go.mod

```diff
@@ -7,8 +7,7 @@ require (
 	github.com/aws/aws-sdk-go v1.28.2
 	github.com/containerd/cgroups v0.0.0-20190919134610-bf292b21730f
 	github.com/containernetworking/cni v0.8.0 // indirect
-	github.com/containers/buildah v1.17.0 // indirect
-	github.com/containers/libpod/v2 v2.0.4
+	github.com/containers/podman/v2 v2.2.1
```

Comment:
Use most recent 'libpod' which is now 'podman' there’s a redirection, and the lastest version is declared as 'github.com/containers/podman' therefor the need for name change.